.buildkite/pipelines/performance/data_set_extraction_daily.yml

```diff
@@ -0,0 +1,43 @@
+steps:
+  - label: ':male-mechanic::skin-tone-2: Pre-Build'
+    command: .buildkite/scripts/lifecycle/pre_build.sh
+    agents:
+      queue: kibana-default
+    timeout_in_minutes: 10
+
+  - wait
+
+  - label: ':building_construction: Build Kibana Distribution and Plugins'
+    command: .buildkite/scripts/steps/build_kibana.sh
+    agents:
+      queue: c2-16
+    key: build
+    if: "build.env('KIBANA_BUILD_ID') == null || build.env('KIBANA_BUILD_ID') == ''"
+
+  - label: ':kibana: Performance Tests with Playwright config'
+    command: .buildkite/scripts/steps/functional/performance_playwright.sh
+    agents:
+      queue: n2-2
```

Comment:
Since we do not track performance in this pipeline but simply wanna run the tests and later fetch APM traces, I wonder what is the best agent to use. Thoughts?

Reply:
For the selenium based tests we use `n2-4-spot`.  I believe n2-2 was attempted at one point, but can't find the discussion.  We can try `n2-2-spot` and see how it goes, and upsize if needed, or the reverse with `n2-4-spot`.  We do collect memory and cpu metrics

Reply:
changed to `n2-4-spot`, I will test this branch after merging pipeline PR